geomesa-hbase/geomesa-hbase-datastore/src/main/scala/org/locationtech/geomesa/hbase/data/HBaseDataStoreFactory.scala

```diff
@@ -24,15 +24,17 @@ class HBaseDataStoreFactory extends DataStoreFactorySpi {
 
   import HBaseDataStoreFactory.Params._
 
+  // TODO: investigate multiple HBase connections per jvm
+  private lazy val globalConnection = ConnectionFactory.createConnection(HBaseConfiguration.create())
+
   // this is a pass-through required of the ancestor interface
-  override def createNewDataStore(params: java.util.Map[String, Serializable]) = createDataStore(params)
+  override def createNewDataStore(params: java.util.Map[String, Serializable]): DataStore = createDataStore(params)
 
   override def createDataStore(params: java.util.Map[String, Serializable]): DataStore = {
     import GeoMesaDataStoreFactory.RichParam
 
-    val connection = ConnectionParam.lookupOpt[Connection](params).getOrElse {
-      ConnectionFactory.createConnection(HBaseConfiguration.create())
-    }
+    val connection = ConnectionParam.lookupOpt[Connection](params).getOrElse(globalConnection)
```

Comment:
this connection will still get closed by a `datastore.dispose()`, right? seems like we'd want to create a new one per data store